src/graph.py

```diff
@@ -0,0 +1,275 @@
+from time import sleep, time_ns
+
+import ccxt  # type: ignore
+
+
+class PricePath:
+    def __init__(self, exchanges: list = [], gdict: dict = {}, cache: dict = {}):
```

Comment:
eh no^^ i just thought due to us only initializing once this would be a non issue.
my idea was (looking backwards maybe not the best) to save the cache or data to the disk so on the next run it has not to be loaded again, but that would be much simpler with a dedicated function for that.
also loading/storing timerange data on the disk has the potential drawback that when an exchange drops historical data (unlikely but possible) that our information we have on disk may not be accurate. also we would need to update the pairdata each load anyway (well at least until the last entry in the operations) which will be the same cost as fetching the total timerange